modules/core/src/main/scala/org/tessellation/l0/domain/snapshot/SnapshotService.scala

```diff
@@ -1,16 +1,10 @@
 package org.tessellation.l0.domain.snapshot
-import cats.data.NonEmptySet
-
 import org.tessellation.dag.domain.block.DAGBlock
-import org.tessellation.kernel.StateChannelSnapshot
-import org.tessellation.schema.peer.PeerId
-import org.tessellation.security.Hashed
+import org.tessellation.security.signature.Signed
 
 trait SnapshotService[F[_]] {
 
-  def createSnapshot(
-    blocks: NonEmptySet[Hashed[DAGBlock]],
-    snapshots: Set[StateChannelSnapshot],
-    nextFacilitators: NonEmptySet[PeerId]
-  ): F[GlobalSnapshot]
+  def offer(block: Signed[DAGBlock]): F[Unit]
+  def createEmptySnapshot(): F[GlobalSnapshot]
+  def createNormalSnapshot(blocks: Set[Signed[DAGBlock]]): F[GlobalSnapshot]
```

Comment:
if that's possible I'd stop naming it `normal` snapshot. Just a snapshot:
```scala
def createEmptySnapshot(): ...
def createSnapshot(): ...
```
or maybe even better:
```scala
def createSnapshot(): F[GlobalSnapshot]
def createSnapshot(blocks: ...): F[GlobalSnapshot]
```

Reply:
Yeah, I agree